utils/file.js

```diff
@@ -103,6 +107,27 @@ exports.fileSystem = (function(){
 
     addShardsToManifest(manifest, file, manifestName, dir, callback);
   }
+  const copyShards = (chunk, chunkId, manifest) => {
+    const copyNum = 3;
+    let copyShardContent;
+    let appendBytes;
+
+    for (let i = 1; i <= copyNum; i++) {
+      appendBytes = crypto.randomBytes(2).toString('hex');
+      copyShardContent = chunk + appendBytes;
+      // console.log(`Copy chunk has ${copyShardContent.length} bytes of data.`);
+
+      const copyChunkId = sha1HashData(copyShardContent);
+      manifest.chunks[chunkId].push(copyChunkId);
+    }
+
+    // console.log(chunkId + ": " + manifest[chunkId]);
```

Comment:
Should we remove these commented out lines?

Reply:
We can remove